Revit_UI/RevitListener.cs

```diff
@@ -144,8 +144,7 @@ public Result OnStartup(UIControlledApplication uIControlledApplication)
 
             //Make sure all BHoM assemblies are loaded
             string versionNumber = uIControlledApplication.ControlledApplication.VersionNumber;
-            string path = Environment.GetEnvironmentVariable("APPDATA") + @"\Autodesk\Revit\Addins\" + versionNumber +  @"\BHoM";
-            BH.Engine.Reflection.Compute.LoadAllAssemblies(path);
+            BH.Engine.Reflection.Compute.LoadAllAssemblies();
```

Comment:
very very minor, but think we could remove the versionNumber string here now as not used any longer:)